src/mongo/db/mongod_options_encryption.idl

```diff
@@ -110,3 +110,42 @@ configs:
         arg_vartype: Switch
         requires: 'security.vault.serverName'
 
+    'security.kmip.serverName':
+        description: 'Hostname or IP address of the KMIP server'
+        short_name: kmipServerName
+        arg_vartype: String
+        requires: [ 'security.enableEncryption', 'security.kmip.port', 'security.kmip.serverCAFile', 'security.kmip.clientCertificateFile', 'security.kmip.clientKeyFile', 'security.kmip.keyIdentifier' ]
```

Comment:
"Port" could be defaulted, and it is defaulted on mongo enterprise. But I left it as required, because the vault keyring also requires the user to specify the port manually.

I could also default the cert files to an empty string, that way if a user has a certificate chain set up correctly on the machine, it can be left out. But most of the time that won't be the case, and this way it's not that easy to forget them.